utilities/merge_operators/cassandra/serialize.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * Helper functions which serialize and deserialize integers
+ * into bytes in big endian.
+ * @author Chen Shen (chenshen@fb.com)
+ * Copyright 2017 Facebook
+ */
+#pragma once
+
+namespace rocksdb {
+namespace cassandra {
+namespace {
+const int64_t kCharMask = 0xFFLL;
+}
+
+template<typename T>
+void Serialize(T val, std::string* dest);
+
+template<typename T>
+T Deserialize(const char* src, std::size_t offset=0);
+
+// Specializations
+template<>
+inline void Serialize<int8_t>(int8_t t, std::string* dest) {
+  dest->append(1, static_cast<char>(t & kCharMask));
+}
+
+template<>
+inline void Serialize<int32_t>(int32_t t, std::string* dest) {
+  for (int i = 0; i < 4; i++) {
```

Comment:
In this class there are many constant numbers used, such as `4`, `8`, `3` and `7`. Could I guess that might be better as `sizeof(int32_t)`, `sizeof(int64_t)`, `sizeof(int32_t) - 1` etc